dompdf/dompdf/CVE-2023-50262.yaml

```diff
@@ -0,0 +1,8 @@
+title: 'Denial of service caused by infinite recursion when parsing SVG images'
```

Comment:
"Resource exhaustion caused by infinite recursion when validating SVG images" (according to https://github.com/dompdf/dompdf/security/advisories/GHSA-3qx2-6f78-w2j2)?

Reply:
Seems that this issue has two different titles: https://github.com/advisories/GHSA-3qx2-6f78-w2j2

One title describes the cause, second the effect.